pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -220,6 +220,9 @@ spec:
               mountPath: /etc/ceph-csi-config/
             - name: keys-tmp-dir
               mountPath: /tmp/csi/keys
+            - name: oidc-token
+              mountPath: /var/run/secrets/tokens
```

Comment:
I think it's better to use `/run` instead since in FHS 3.0, `/var/run` is replaced by `/run`.

Reply:
@leseb cephcsi expects it to be in this hard-coded part? any specific reason to replace this one. or else we need code changes in cephcsi to change the hardcoded value https://github.com/ceph/ceph-csi/blob/6f1c276dc83eab0f74e1620d47821dedd5172267/internal/kms/aws_sts_metadata.go#L60-L63

Reply:
Well I think it's good to stick with the latest standard. Is it possible to get the code change in cephcsi?

Reply:
@Rakshith-R can we get changes in cephcsi for this one also?

Reply:
https://github.com/ceph/ceph-csi/pull/2959 
done